exonum/src/runtime/execution_context.rs

```diff
@@ -184,7 +184,7 @@ impl<'a> ExecutionContext<'a> {
     }
 
     /// Re-borrows an execution context with the given instance descriptor.
-    fn reborrow<'s>(&'s mut self, instance: InstanceDescriptor<'s>) -> ExecutionContext<'s> {
+    fn reborrow(&mut self, instance: InstanceDescriptor) -> ExecutionContext {
```

Comment:
Use `-> ExecutionContext<'_>`; the child context still has lifetime dependent on `&mut self`.